### DISCUSSION TITLE
My first performance file was a failure

### DISCUSSION THREAD

**Discussion by Alan-Angel** [probonopd/MiniDexed#600] [General]
So I basically wanted 4 TGs to work on one half of my keyboard and 4 TGs to work on the other half. I copied a voice from one performance into another, it all looks good and TG 1-4 works as expected but TGs 5-8 are completely muted. I still don't get the voice system if I'm honest, is there issue because I copied it in? Example of TGs 4 and 5 because 4 works and 5 doesn't. Thanks in advance!:

```
# TG4
BankNumber4=0
VoiceNumber4=1
MIDIChannel4=1
Volume4=30
Pan4=127
Detune4=-2
Cutoff4=99
Resonance4=0
NoteLimitLow4=0
NoteLimitHigh4=59
NoteShift4=0
ReverbSend4=99
PitchBendRange4=12
PitchBendStep4=0
PortamentoMode4=0
PortamentoGlissando4=0
PortamentoTime4=0
VoiceData4=39 0B 09 1B 57 63 5A 00 27 00 00 00 00 00 00 04 45 00 03 00 04 21 63 63 1B 63 63 63 00 30 00 09 00 00 00 00 00 46 00 01 00 07 46 2C 27 39 5D 63 00 00 27 00 00 00 00 00 00 01 4B 00 02 00 0E 39 18 0C 23 57 63 5D 00 27 00 00 00 00 00 00 01 63 00 02 00 07 40 14 0C 1B 5D 63 5D 00 21 06 00 00 00 00 00 04 50 00 01 00 03 3C 18 63 26 5A 63 63 00 27 00 00 00 00 00 00 01 63 00 01 00 07 63 63 63 63 32 32 32 32 08 07 00 1D 00 09 00 00 00 03 18 53 6B 79 4C 61 62 20 20 20 20 00
MonoMode4=0
ModulationWheelRange4=99
ModulationWheelTarget4=1
FootControlRange4=99
FootControlTarget4=0
BreathControlRange4=99
BreathControlTarget4=0
AfterTouchRange4=99
AfterTouchTarget4=0

# TG5
BankNumber5=0
VoiceNumber5=1
MIDIChannel5=1
Volume5=58
Pan5=0
Detune5=-3
Cutoff5=99
Resonance5=0
NoteLimitLow5=60
NoteLimitHigh5=127
NoteShift5=244
ReverbSend5=99
PitchBendRange5=2
PitchBendStep5=0
PortamentoMode5=0
PortamentoGlissando5=0
PortamentoTime5=0
VoiceData5=60 4B 0F 14 54 63 60 00 30 00 09 00 00 00 01 00 51 00 01 00 07 4B 3F 1E 0C 63 60 5D 00 33 00 09 00 00 00 00 00 52 00 01 00 07 4E 2D 1F 28 63 60 00 00 00 00 00 00 00 00 00 01 63 00 01 00 07 4E 4B 21 14 54 63 60 00 33 00 09 00 00 00 00 00 4B 00 02 00 0A 63 63 15 0C 63 63 60 00 33 00 06 00 00 00 00 00 53 00 02 00 08 4E 45 22 2E 63 63 00 00 00 00 03 00 00 00 00 02 54 01 00 00 07 63 63 63 54 32 32 32 32 02 07 00 1A 00 0A 15 00 00 02 18 50 6F 6C 79 53 79 6E 74 68 20 3F
MonoMode5=1
ModulationWheelRange5=99
ModulationWheelTarget5=1
FootControlRange5=99
FootControlTarget5=0
BreathControlRange5=99
BreathControlTarget5=0
AfterTouchRange5=99
AfterTouchTarget5=0
```

**Comment by Banana71** [comment]
Hi Alan,

This is a Split-Sound

TG4 is in the left Side and TG5 is on the right Side
NoteLimitLow4=0
NoteLimitHigh4=59

NoteLimitLow5=60
NoteLimitHigh5=127

Set all NotelimitLow to 0 and all NotelimitHigh to 127

**Reply by Alan-Angel** [reply]
Hi Banana71, I want them to be split before and after C3, is that not what the note limits are for?

**Reply by Banana71** [reply]
You're right about the split point.  I misunderstood your question.  
Why did you set Noteshift5 to 244?  Probably only bats can hear the sound 😂

**Reply by Alan-Angel** [reply]
Aha! Yes the Minidexed CC editor did that for some reason, the notes we're just too high to hear. You'd think my dog would have told me. Maybe someone else can replicate the bug? I got it to work properly once on one TG but the rest of the time it's throwing in crazy values. Is it putting in a slider range of 0-255 instead of =/-24 semitones I wonder?
